### DISCUSSION TITLE
2022-01-21 Infrastructure Update

### DISCUSSION THREAD

**Discussion by rtsai123** [near/community#175] [Infrastructure]
DPI Team Items
- Configure and enable Google Translator to gov.near.org https://github.com/near/near-ops/issues/759
- Debugging indexer state viewer
- Provisioned  RPC  and archival TESTNET/MAINNET nodes with larger disks  https://github.com/near/near-ops/issues/710
- Fixed snapshot related issues https://github.com/near/nearcore/issues/6153
- Debugged(RPC) Bowen testing machine mainnet archive https://github.com/near/near-ops/pull/754
- Finalized OKRs and populated to JIRA OKR board
- Performance reviews 
- Migrating from Notion Site Reliability content to Confluence 
- Established Dev Platform Infrastructure team JIRA project scrum board 
- SPIKE with Redis Enterprise Vendor right sizing for FastRPC/state-indexer/throttler for global active-active solution

**Comment by jberrytech** [comment]
I want to welcome @rtsai123 as the head of the Infrastructure team! Excited to have you on the team, Robert!
